sky/unit/test/gestures/scale_test.dart

```diff
@@ -36,7 +36,7 @@ void main() {
 
     TestPointer pointer1 = new TestPointer(1);
 
-    ui.PointerEvent down = pointer1.down(new Point(10.0, 10.0));
+    PointerInputEvent down = pointer1.down(new ui.Point(10.0, 10.0));
```

Comment:
I was trying to fix warnings that I thought were related to my merge, but I may have gotten overzealous and fixed warnings that were unrelated.  The test passes with this change, I'm not 100% sure whether it passes without it - I can revert and see.